app/views/rounds/_pairings.html.slim

```diff
@@ -1,5 +1,10 @@
 - single_sided = round.stage.single_sided?
 - round.pairings.each do |pairing|
+  - left_player = @players[pairing.player1_id]
+  - right_player = @players[pairing.player2_id]
+  - if single_sided && !pairing.player1_is_corp?
+    - left_player = @players[pairing.player2_id]
+    - right_player = @players[pairing.player1_id]
```

Comment:
For a top cut, putting the corp first seems to lose information? I think who comes first is based on their position in the bracket?

Reply:
The top cut isn't showing corp player first - this behavior seems unchanged from looking at a cut.